Library/Homebrew/cmd/upgrade.rb

```diff
@@ -126,6 +127,9 @@ def upgrade_args
   def upgrade
     args = upgrade_args.parse
 
+    # Deprecated since this is now the default behavior.
+    # odeprecated "`brew upgrade --ignore-pinned`" if args.ignore_pinned?
```

Comment:
```suggestion
    odeprecated "`brew upgrade --ignore-pinned`" if args.ignore_pinned?
```

Deprecating this immediately as new tag will be a minor release 🎉